projects/aca-playwright-shared/src/api/file-actions.ts

```diff
@@ -50,6 +50,27 @@ export class FileActionsApi {
     });
   }
 
+  async uploadFileWithRename(fileLocation: string, parentId: string = '-my-', newName: string, title: string = '', description: string = '') {
+    const file = fs.createReadStream(fileLocation);
+    const nodeProps = {
```

Comment:
I think you can just apply `NodeBodyCreate` type here to make sure you're adding all required params, instead of adding `as NodeBodyCreate` in the return statement

Reply:
added